src/Compilers/CSharp/Test/IOperation/IOperation/IOperationTests_IArrayElementReferenceExpression.cs

```diff
@@ -519,9 +519,16 @@ public void F()
 ";
 
             string expectedOperationTree = @"
-IInvalidOperation (OperationKind.Invalid, Type: ?) (Syntax: '[0]')
-  Children(0)";
-            var expectedDiagnostics = new DiagnosticDescription[] { };
+IInvalidOperation (OperationKind.Invalid, Type: null, IsInvalid) (Syntax: '[0]')
+  Children(1):
+      ILiteralOperation (OperationKind.Literal, Type: System.Int32, Constant: 0, IsInvalid) (Syntax: '0')
+";
+            var expectedDiagnostics = new DiagnosticDescription[]
+            {
+                // (6,13): error CS0815: Cannot assign collection literals to an implicitly-typed variable
```

Comment:
This error will go away eventually, right?

Reply:
Yes, this error should disappear when support for natural type is added.